src/Pact/Types/Crypto.hs

```diff
@@ -219,64 +167,59 @@ instance ConvertBS Ed25519.Signature where
   fromBS = Right . Ed25519.Sig
 #endif
 
-
```

Comment:
We're talking more about ETH support these days, and especially since `Scheme` got so trimmed down, I don't think we should remove support anymore. Especially with the cleanups in `ApiReq` and `Command` for making commands, it's my belief that it should be a lot easier to support now?